### PR TITLE
[SHARE-338][Feature] Add Cyprus University of Technology OAI provider

### PR DESCRIPTION
**Pull request by aaxelb**
https://openscience.atlassian.net/browse/SHARE-338

### CODE REVIEW

**Comment by leb2dg** [comment]
It seems like their feed isn't working again?

File: project/settings.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
     'providers.ca.umontreal',
     'providers.ca.uwo',
     'providers.ch.cern',
+    'providers.cy.ac.cut',
```
Comment (leb2dg) [inline project/settings.py]: Would it still be accurate to remove the `ac` directory so that there isn't an extra folder involved?

**Comment by chrisseto** [comment]
@aaxelb should this be marked as on hold?

**Comment by aaxelb** [comment]
@chrisseto It is on hold in JIRA, should I put it in the PR title, too?

**Comment by aaxelb** [comment]
Closed in favor of #608

**Pull request closed without merging**